src/transformers/models/pegasus/tokenization_pegasus.py

```diff
@@ -250,35 +249,10 @@ def build_inputs_with_special_tokens(self, token_ids_0, token_ids_1=None) -> Lis
         # We don't expect to process pairs, but leave the pair logic for API consistency
         return token_ids_0 + token_ids_1 + [self.eos_token_id]
 
-    @add_start_docstrings(PREPARE_SEQ2SEQ_BATCH_DOCSTRING)
-    def prepare_seq2seq_batch(
-        self,
-        src_texts: List[str],
-        tgt_texts: Optional[List[str]] = None,
-        max_length: Optional[int] = None,
-        max_target_length: Optional[int] = None,
-        return_tensors: str = None,
-        truncation=True,
-        padding="longest",
-        **unused,
-    ) -> BatchEncoding:
+    def prepare_seq2seq_batch(self, src_texts: List[str], **kwargs) -> BatchEncoding:
         if "" in src_texts:
             raise ValueError(f"found empty string in src_texts: {src_texts}")
```

Comment:
I don't know if this check is important. It should be removed IMO, there is no reason for a hard failure if one text is empty. It may have been linked to the bug where the pad method failed if the first text was empty, but this bug is now fixed.

If we remove the check, there is no need to override the method.